README.md

```diff
@@ -48,7 +48,9 @@ Output team
 
 ### Generating a gatorgrade.yml file
 
-Generate team
+The generation of yml file can be used to configure tests.
+
+In the terminal within the main directory, use the command `gatorgrade generate <path> <root directory>` to generate a yml file.
```

Comment:
Since this PR is about adding tests, I would save these changes for another PR. Additionally, I'm not sure the wording here makes sense either -- I don't think the `generate` command is being used to configure tests, but rather create assignments.